mail/mutt_ics/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                mutt_ics
+version             0.9.2
+license             MIT
+categories          mail python
+platforms           darwin
+homepage            https://github.com/dmedvinsky/mutt-ics
+maintainers         {@gpanders gpanders.com:greg} \
+                    openmaintainer
+
+description         A tool to show calendar event details in Mutt
+long_description    mutt_ics allows you to view calendar attachments inline \
+                    in Mutt.
+
+python.default_version \
+                    38
```

Comment:
We're defaulting to Python 3.9 these days - any particular reason for 3.8?

Reply:
Only that `py-icalendar` only supports up to 3.8. I can update the icalendar port as well and include that in this PR and have them both use 3.9.

Reply:
Updated py-icalendar and mutt_ics to use Python 3.9